kolibri/deployment/default/settings/base.py

```diff
@@ -86,6 +87,10 @@
     }
 }
 
+# DIR for storing contentDBs
+CONTENT_DB_DIR = os.path.abspath(os.path.join(__file__, "../../../..")) + "/content/content_db"
```

Comment:
Use `os.path.join`... if this is package data, use pkgutil to resolve it: http://stackoverflow.com/questions/5897666/how-do-i-use-data-in-package-data-from-source-code

But if it's user data, don't use these patterns at all, use KOLIBRI_HOME